trade-finance-manager-api/src/types/db-models/tfm-users.ts

```diff
@@ -2,16 +2,13 @@ import { WithId } from 'mongodb';
 import { TeamId } from '../team-id';
 
 export type TfmUser = WithId<{
```

Comment:
This is different to TfmUser in our MongoDb model. Are we storing azureOid in the database model? We should differenciate these

Reply:
This is absolutely out of sync. It should be removed from here and dtfs-central should be made in line with this.